src/test/splitting/tasks_tests_handler/previous_errors_test.py

```diff
@@ -1,26 +1,29 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
 import os
+import pytest
```

Comment:
Pytest is not a standard library, so please, move this import into a separated second group